portal/components/react-admin/layout/customAppBar.js

```diff
@@ -44,8 +42,8 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const AppBarCustom = (props) => {
-  const [session, loading] = useSession();
+const AppBarCustom = () => {
```

Comment:
Keep consistent either use fat arrow function or a function keyword in all files.

Reply:
okay